website/blog/2020-01-10-cobalt.md

```diff
@@ -241,13 +239,6 @@ the user the option of automatic reconnection. This works both when the build
 server was down and when the connection was just interrupted. It should also
 solve the issues on MacOs with dropped connection.
 
-### Experimental build tool support for Pants
-
-Thanks to the recent work by [@olafurpg](https://github.com/olafurpg), there's
-now experimental support for the Pants build tool. The Pants integration is
-still evolving so it's not yet documented. The end goal is to support Pants in
-the same way as other build tools like sbt, Maven and Gradle.
-
```

Comment:
Since these blogs somewhat serve as release notes and _sort of_ a changelog, do we really want to change this blog post to remove reference at all to Pants? This seems to imply that we are sort of _hiding_ away that we had experimental pants support at one time. My question is mainly just trying to understand the reason behind this.

Reply:
My reasoning to remove this part is because it appears when users search for "Pants" on the website. Since the Pants integration is now gone I figured it would be best if this section didn't show up when people search for it.